hypervisor.go

```diff
@@ -154,6 +154,10 @@ type HypervisorConfig struct {
 	// DisableBlockDeviceUse disallows a block device from being used.
 	DisableBlockDeviceUse bool
 
+	// BlockDeviceDriver specifies the driver to be used for block device
+	// either "virtio-scsi" or "virtio-blk" with the default driver being "virtio-scsi"
```

Comment:
For maintenance, it's going to be safer if you refer to `VirtioBlock` and `VirtioSCSI`. But It could also be useful to define a `DefaultVirtio` which could also be mentioned here.

Reply:
Fixed.